### PR TITLE
Receiver v4

### PR DESCRIPTION
**Pull request by marcosmx**
This branch has the needed changes to implement a chromecast app recevier by using OOyala's cast receiver library.

### CODE REVIEW

File: receiver.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <script type="text/javascript" src="//player.ooyala.com/core/e48c9b51282d406f957c666149763424?plugins=bm,cast"></script>    
+</head>
+<body>
+    <div id="splash-screen" class="fbox-container fs">
+        <img id="splash-image" src="images/logo.svg" />
+        <div id="status-cast">
+        </div>
+    </div>
+    <div id="player"></div>
+<script type="text/javascript" src="//static.ooyala.com/chromecast/latest/cast.min.js"></script></body>
```
Comment (aldigun) [inline receiver.html]: This URL is not correct. You have to use
//player.ooyala.com/static/chromecast/latest/cast.min.js

File: README.md [hunk 2]
```diff
@@ -1,41 +1,33 @@
-# chromecast-sample-receiver
-This is the repository that contains an open-source Ooyala Chromecast sample receiver. This application should not be used for testing or evaluating Ooyala capabilities.
+# Chromecast Receiver + OOyala Player v4
```
Comment (pilievOoyala) [inline README.md]: OOyala -> Ooyala

File: README.md [hunk 3]
```diff
@@ -1,41 +1,33 @@
...
+This repository contains the implementation of Google cast sdk as a receiver for video playback through Chromecast devices. It has an adaptation of OOyala Player v4.
```
Comment (pilievOoyala) [inline README.md]: OOyala -> Ooyala

Comment (pilievOoyala) [reply README.md]: There are a couple more places below too

File: receiver.html (hunk 1 above)
Comment (marcosmx) [reply receiver.html]: In that case I need to ask access to player.ooyala.com/static/ on the s3 buckets

File: README.md (hunk 2 above)
Comment (marcosmx) [reply README.md]: that block was removed

File: README.md (hunk 3 above)
Comment (marcosmx) [reply README.md]: the block was removed

**Comment by pilievOoyala** [comment]
Looks good to me other than misspelling the company name.

**Comment by aldigun** [comment]
LGTM.

**Pull request merged**